docs/source/posts/2023/2023-07-24-week-8-joaodellagli.rst

```diff
@@ -0,0 +1,119 @@
+My new task was, as I had this example working, to start the API development. In a meeting with Bruno, one of my mentors, we debated 
+on how could this work, reaching two options:
+
+1. Implement the KDE in a single, simple actor.
+2. Implement a KDE rendering manager, as a class.
+
+The first one would have the advantage of being simple and pretty straightforward, as a user would only need to call the actor and have 
+it working on their hands, having the tradeoff of leaving some important steps for a clean API hidden and static. These steps I mention 
+are related to how this rendering works, as I have previously :doc:`showed you <2023-07-03-week-5-joaodellagli>`, it relies on post-processing effects, 
+which need an offscreen rendering, that for example are done by the *callback functions*. 
+
+In short, these functions are instructions the user gives to the interactor to run inside the interaction loop. Inside FURY there are tree
+types of callbacks passed to the window interactor:
+
+1. **Timer Callbacks**: Added to the window interactor, they are a set of instructions that will be called from time to time, with interval defined by the user.
+2. **Window Callbacks**: Added directly to the window, they are  a set of instructions called whenever an specific event is triggered.
+3. **Interactor Callbacks**: Added to the window interactor, they are a set of instructions called whenever an specific interaction, for example a mouse left-click, is triggered. 
+
+In this API, I will be using the *Interactor Callback*, set by the ``window.add_iren_callback()`` function, that will be called whenever a *Render* 
+interaction is detected, and needs to be first passed to the onscreen manager. 
+
+These details are more complicated, and would need, for example, for the user to pass the onscreen manager to the ``actor.kde()`` function. 
+Also, in the case of a kde actor not being used anymore and being declared, the callback then passed would still exist inside the manager and
+be called even when the kde actor is not on screen anymore, which is not ideal.
+
+Knowing these problems, we thought of a second option, that would have the advantage of not leaving those details and steps behind. It has 
+the tradeoff of maybe complicating things as it would need to be called after calling the effects manager, but as I will show you below, 
+it is not that complicated *at all*. 
+
+I also reviewed my fellow GSoC contributors PR's as well, PR `#810 <https://github.com/fury-gl/fury/pull/810>`_ and 
+`#803 <https://github.com/fury-gl/fury/pull/803>`_. Bruno told me to take a look as well on `Conventional Commits <https://www.conventionalcommits.org>`_ , a way to standardize
+commits by prefixes, so I did that as well.
+
+So how did it go?
+-----------------
+
+Well, the implemented manager class is named ``EffectManager()`` and to initialize it you only need to pass the onscreen manager:
+
+.. code-block:: python
+
+   effects = EffectManager(onscreen_manager)
```

Comment:
What is `onscreen_manager` ? you do not talk about it before. Maybe just add one line of code to show how you initialize this object.